_el/exercises.md

```diff
@@ -1,168 +1,95 @@
-> *not* in that directory?
-> 
+Κάθε καλός ξυλουργός έχει μια σειρά από κατσαβίδια και κάθε καλός δάσκαλος έχει διαφορετικά είδη ασκήσεων διαμορφωτικής αξιολόγησης τα οποία μπορεί να χρησιμοποιήσει για να ελέγχει τι μαθαίνουν πραγματικά οι μαθητές, να τους βοηθά να εξασκούν τις νέες τους δεξιότητες, και να διατηρεί ζωντανό το ενδιαφέρον τους. Αυτό το κεφάλαιο αρχίζει περιγράφοντας διάφορα είδη ασκήσεων που μπορείτε να χρησιμοποιήσετε για να ελέγξετε αν η διδασκαλία σας ήταν αποτελεσματική. Στη συνέχεια κάνει μια επισκόπηση της τρέχουσας κατάστασης στην τεχνολογία της αυτοματοποιημένης διόρθωσης και καταλήγει σε μια διερεύνηση των συζητήσεων, των έργων, και των άλλων σημαντικών ειδών εργασίας τα οποία απαιτούν περισσότερη ανθρώπινη προσοχή για να αξιολογηθούν. Η συζήτησή μας βασίζεται εν μέρει στην [Τράπεζα Ερωτήσεων του Canterbury] (http://web-cat.org/questionbank/)[[Sand2013] (# CITE)], η οποία περιέχει καταχωρήσεις για διάφορες γλώσσες προγραμματισμού και θέματα που αφορούν την εισαγωγικού επιπέδου υπολογιστική.
+
+## Οι κλασσικές {#s: ασκήσεις-κλασικά}
+
+Όπως συζητήθηκε στην Ενότητα [s:models-formative-assessment](#SECTION), οι *ερωτήσεις πολλαπλών επιλογών* (ΕΠΕ) είναι πιο αποτελεσματικές όταν οι λανθασμένες απαντήσεις παραπέμπουν σε συγκεκριμένες παρανοήσεις. Σύμφωνα με την ταξινόμηση του Bloom ([s:process-objectives](#SECTION)), οι ΕΠΕ συνήθως σχεδιάζονται για να ελέγξουν την ανάκληση και την κατανόηση ενός θέματος ("Ποια είναι η πρωτεύουσα του νομού Πέλλας;"), αλλά μπορούν επίσης να απαιτούν από τους εκπαιδευόμενους να ασκούν και τις κριτικές τους ικανότητες.
+
+> ** Μια ερώτηση πολλαπλής επιλογής **
+>
+> Με ποιά σειρά γίνονται οι υπολογισμοί όταν ο υπολογιστής κάνει αποτίμηση της έκφρασης `price = addTaxes(cost - discount)`;
+>
+> 1. αφαίρεση, κλήση συνάρτησης, ανάθεση
+> 1. κλήση συνάρτησης, αφαίρεση, ανάθεση
+> 1. κλήση συνάρτησης, μετά ανάθεση και αφαίρεση ταυτόχρονα
+> 1. κανένα από τα παραπάνω
+
+Ένας δεύτερος κλασικός τύπος άσκησης προγραμματισμού είναι ο *κώδικας και εκτέλεση* (Κ&Ε), στον οποίο ο εκπαιδευόμενος γράφει κώδικα ο οποίος παράγει ένα καθορισμένο αποτέλεσμα. Οι ασκήσεις Κ&Ε μπορούν να είναι από πολύ απλές έως αρκετά περίπλοκες ανάλογα με τις ανάγκες του δασκάλου, αλλά για την ορθή χρήση τους εντός μιας αίθουσας διδασκαλίας, θα πρέπει να είναι σύντομες και να έχουν μόνο μία ή δύο πιθανώς σωστές απαντήσεις. Στους αρχάριους, είναι συχνά αρκετό να τους ζητήσετε να καλέσουν μια συγκεκριμένη συνάρτηση: οι έμπειροι εκπαιδευτικοί έχουν την τάση να ξεχνούν ότι μπορεί κάποιοι να βρίσκουν δύσκολη την έννοια της τοποθέτησης παραμέτρων στη σωστή σειρά. Για τους πιο προχωρημένους μαθητές, το να βρουν ποια συνάρτηση πρέπει να καλέσουν είναι συχνά πιο ελκυστικό και επομένως και καλύτερη σημάδι (μετρική) της κατανόησής τους.
+
+> **Κώδικας & Εκτέλεση**
+>
+> Η μεταβλητή `picture` περιέχει μια έγχρωμη εικόνα που διαβάζεται από ένα αρχείο.
+> Χρησιμοποιώντας μια λειτουργία, δημιουργήστε μια ασπρόμαυρη έκδοση της εικόνας και
```

Comment:
λειτουργια -- > συνάρτηση

Reply:
Thanks @pdelias ! Fixed in a new commit.